content/influxdb/v2.0/write-data/developer-tools/csv.md

```diff
@@ -226,6 +226,8 @@ influx write -b example-bucket \
 Skipped rows are ignored and are not written to InfluxDB.
 {{% /warn %}}
 
+Use the `--error-file` flag to record errors to a file.
```

Comment:
Perhaps we can add example that describe how this error file contains all rows that can't be imported and includes comments about what is wrong, for example:

# error : line 3: column 'a': '1.1' cannot fit into long data type
cpu,1.1